src/components/list/themes/light/list-item.bootstrap.scss

```diff
@@ -0,0 +1,22 @@
+@use './list-item.base.scss';
```

Comment:
This should rather be imported and used as the default static styles in the list-item.ts. It shouldn't be imported here.